source/expressions.tex

```diff
@@ -4155,11 +4155,13 @@
 If the operand is a \grammarterm{qualified-id} naming a non-static or variant member \tcode{m}
 of some class \tcode{C} with type \tcode{T}, the result has type ``pointer to member
 of class \tcode{C} of type \tcode{T}'' and is a prvalue designating \tcode{C::m}.
-Otherwise, if the type of the expression is \tcode{T}, the result has type ``pointer to
-\tcode{T}'' and is a prvalue that is the address of the designated object\iref{intro.memory}
-or a pointer to the designated function. \begin{note} In particular, the address of an
-object of type ``\cv{}~\tcode{T}'' is ``pointer to \cv{}~\tcode{T}'', with the same
-cv-qualification. \end{note}
+Otherwise, if the type of the expression is \tcode{T},
+the result is a prvalue of type ``pointer to \tcode{T}''
+whose value is a pointer to the designated object\iref{intro.memory} or function.
+\begin{note}
+In particular, taking the address of a variable of type ``\cv{}~\tcode{T}''
+yields a pointer of type ``pointer to \cv{}~\tcode{T}''.
+\end{note}
```

Comment:
I think this note is extraneous, but I suspect the editorial telecon already considered eliminating it and chose not to do so.